### ISSUE TITLE
[Feature Request] Incorporate External Sounds Script feature into 4chanX

### ISSUE THREAD

**Issue by oznerol10** [ccd0/4chan-x#2604]
There's been a popular script being passed around on greasyfork recently called 4chan External Sounds. If the creator gave permission, would you be open to incorporating this functionality into 4chanX itself?

https://greasyfork.org/en/scripts/31045-4chan-external-sounds/code.

**Comment by ccd0** [comment]
I don't think this is something I would consider within the scope of 4chan X. And if I did implement this feature, it wouldn't be the version that bends the knee.

**Comment by mid-kid** [comment]
I'm not sure what your vision of scope is for 4chanx, but the main reason I'd consider this is that it's a universally applicable improvement to the site that people won't care about until they need it. Having it work without people having to consciously install a separate script for it will significantly increase the reach of the feature, and incentivize more people to use it.
